packages/core/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -85,7 +86,12 @@ class PersonalDetails extends React.Component {
     };
 
     handleValidate = values => {
-        const { errors, warnings } = splitValidationResultTypes(this.props.validate(values));
+        const { warnings, errors } = splitValidationResultTypes(this.props.validate(values));
+        const validateValues = validate(errors, values);
+
+        const only_alphabet_fields = ['first_name', 'last_name'];
+        validateValues(validLetterSymbol, only_alphabet_fields, localize('Only alphabet is allowed'));
+
```

Comment:
I tried to follow the same steps as other forms with validation

Reply:
@shayan-binary in account package they don't have access to the other validations and maybe that's why they're using this validator function? the message `Only alphabet is allowed` is grammatically wrong anyway, where did this even come from in our codebase 😅 we should validate these fields using the `validLetterSymbol` function